src/prefect/engine.py

```diff
@@ -1988,16 +1994,21 @@ def should_log_prints(flow_or_task: Union[Flow, Task]) -> bool:
 
     try:
         flow_run_id = UUID(
-            sys.argv[1] if len(sys.argv) > 1 else os.environ.get("PREFECT__FLOW_RUN_ID")
+            sys.argv[-1]
+            if len(sys.argv) > 1
+            else os.environ.get("PREFECT__FLOW_RUN_ID")
         )
+        ignore_storage = "--ignore-storage" in sys.argv
```

Comment:
😬 I would prefer we don't add flags here. This entrypoint should be as minimal as possible and avoid changing for compatibility across versions. Can you use an environment variable instead and just read it in `enter_flow_run_engine_from_subprocess`?

Reply:
I opted to remove all the engine changes since workers should try to execute flow runs that have storage blocks associated with them. A vast majority of the time is would result in the flow run failing. That's handled within the worker code now.